.go-version

```diff
@@ -1 +1 @@
-1.20.13
\ No newline at end of file
+1.20.14
```

Comment:
shouldn't this be updating to go1.21.x? go1.20 is out of support now and we're ready to bump release branches to go1.21 per https://github.com/kubernetes/release/issues/3076#issuecomment-1556053267

Reply:
we need to build the image for that, it is not ready yet, to be honest i forgot that, will start the build for those

sorry